v-next/hardhat-ignition/src/internal/tasks/deploy.ts

```diff
@@ -197,52 +202,43 @@ const taskDeploy: NewTaskActionFunction<TaskDeployArguments> = async (
 
   const strategyConfig = hre.config.ignition.strategyConfig?.[strategyName];
 
-  try {
-    const result = await deploy({
-      config: hre.config.ignition,
-      provider: connection.provider,
-      executionEventListener,
-      artifactResolver,
-      deploymentDir,
-      ignitionModule: userModule,
-      deploymentParameters: parameters ?? {},
-      accounts,
-      defaultSender: defaultSender === "" ? undefined : defaultSender,
-      strategy: strategyName,
-      strategyConfig,
-      maxFeePerGasLimit:
-        hre.config.networks[connection.networkName]?.ignition.maxFeePerGasLimit,
-      maxPriorityFeePerGas:
-        hre.config.networks[connection.networkName]?.ignition
-          .maxPriorityFeePerGas,
-      gasPrice: hre.config.networks[connection.networkName]?.ignition.gasPrice,
-      disableFeeBumping:
-        hre.config.ignition.disableFeeBumping ??
-        hre.config.networks[connection.networkName]?.ignition.disableFeeBumping,
-    });
-
-    // TODO: HH3 Bring back with the port of hardhat-verify
-    // if (result.type === "SUCCESSFUL_DEPLOYMENT" && verify) {
-    //   console.log("");
-    //   console.log(chalk.bold("Verifying deployed contracts"));
-    //   console.log("");
-
-    //   await hre.run({ scope: "ignition", task: "verify" }, { deploymentId });
-    // }
-
-    if (result.type !== "SUCCESSFUL_DEPLOYMENT") {
-      process.exitCode = 1;
-    }
-
-    return result;
-  } catch (_e) {
-    // Disabled for the alpha release
-    // if (e instanceof IgnitionError && shouldBeHardhatPluginError(e)) {
-    //   throw new HardhatError(HardhatError.ERRORS.IGNITION.INTERNAL_ERROR, e);
```

Comment:
I am interpreting this as: we have decided to no longer distinguish between internal Ignition errors. Instead we will throw HardhatError's directly so we will not bring back this mapping.

Reply:
Correct. I didn't see a way that this logic fit in or was necessary now that we are just throwing HardhatError's for everything ourselves